low-level/framework/utils/utility.py

```diff
@@ -156,6 +158,56 @@ def get_current_time():
         """Returns the time as integer number in seconds since the epoch in UTC."""
         return int(time.time())
 
+    @staticmethod
+    def get_manufacturer_name():
+        """Returns node server manufacturer name."""
+        manufacturer = "NA"
+        cmd = "ipmitool bmc info"
```

Comment:
'ipmitool' command should pre-constructed based on the bmc config and not hardcoded.
@cdeshmukh Please confirm that do we need that change here?

Reply:
True, need to use function which constructs ipmitool command, based on config, to use inband or out-of-band channel

Reply:
Yeah, should be called via ipmi_client.IPMITool class.